src/Picqer/Carriers/SendCloud/Connection.php

```diff
@@ -171,6 +173,11 @@ public function parseResponse(ResponseInterface $response)
             $response->getBody()->rewind();
 
             $responseBody = $response->getBody()->getContents();
+
+            if (strlen($responseBody) > $this->maxResponseSizeInBytes) {
```

Comment:
Bij de response vanuit SendCloud krijg je zo'n grote response (70mb aan json met 500k objectes) dat PHP er gewoon uitklapt. Ik weet zelf alleen niet op welk punt. Is dat echt pas tijdens de json_decode? Of is het al bij het omzetten van de response naar een response object? En dus voor dat je de `maxResponseSizeInBytes` kan checken?